low-level/actuators/impl/generic/realstor_encl.py

```diff
@@ -146,9 +143,12 @@ def perform_request(self, jsonMsg):
         return response
 
     def make_response(self, component_details, component, component_type,
-            resource_id, ctrl_action = None):
-
-        resource_type = "enclosure:{}:{}".format(component, component_type)
+            resource_id, ctrl_action=None, sensor=None):

```

Comment:
>component_type == "platform_sensor":

this can easily serve to know if its sensor, why to add new var to maintain same thing ?

Reply:
Because resource hierarchy of `platform_sensor` is different from any other hw resources.
So to make resource_type, sensor variable is required (line 148). 
```
storage:hw:platform_sensor:voltage
storage:hw:psu
```
Still we can avoid extra variable(sensor) and simplify this.
I will make it in next commit.

Reply:
Please check the new changes on this.